src/controller/python/README.md

```diff
@@ -1,244 +1,255 @@
+   ```
+   cd connectedhomeip
+   ```
 
-## BLE Virtualization on Linux (Optional)
+4. Initialize the git submodules.
 
-If we would like to setup virtual BLE central and peripheral in the same
-machine, then Bluez setup
+   ```
+   git submodule update --init
+   ```
 
-```
-cd third_party/bluez/repo
-./bootstrap
-third_party/bluez/repo/configure --prefix=/usr --mandir=/usr/share/man --sysconfdir=/etc --localstatedir=/var --enable-experimental --with-systemdsystemunitdir=/lib/systemd/system --with-systemduserunitdir=/usr/lib/systemd --enable-deprecated --enable-testing --enable-tools
-make
-```
+5. Build and install the Python CHIP controller.
 
-Note: You also need to install several packages on RPi if you want to build
-bluez
+   ```
+   scripts/build_python.sh -m platform
+   ```
 
-```
-sudo apt-get install libtool m4 automake autotools-dev libudev-dev libical-dev libreadline-dev
-```
+   > Note: To get more details about available build configurations, run the following command:
+   > ```scripts/build_python.sh --help```
 
-Run bluetoothd:
+6. Activate the Python virtual environment.
 
-```
-sudo third_party/bluez/repo/src/bluetoothd --experimental --debug &
-```
+   ```
+   source out/python_env/bin/activate
+   ```
 
-Bring up two virtual ble interface:
+7. Run the Python CHIP controller.
 
-```
-sudo third_party/bluez/repo/emulator/btvirt -L -l2
-```
+   > Running as root is necessary to obtain access to the Bluetooth interface.
 
-You can find the virtual interface by `hciconfig` command:
+   ```
+   sudo out/python_env/bin/chip-device-ctrl
+   ```
 
-```
-$ hciconfig
+   You can also select the Bluetooth LE interface using command line argument:
 
-hci2:	Type: Primary  Bus: Virtual
-	BD Address: 00:AA:01:01:00:24  ACL MTU: 192:1  SCO MTU: 0:0
-	UP RUNNING
-	RX bytes:0 acl:95 sco:0 events:205 errors:0
-	TX bytes:2691 acl:95 sco:0 commands:98 errors:0
+   ```
+   sudo out/python_env/bin/chip-device-ctrl --bluetooth-adapter=hci2
+   ```
 
-hci1:	Type: Primary  Bus: Virtual
-	BD Address: 00:AA:01:00:00:23  ACL MTU: 192:1  SCO MTU: 0:0
-	UP RUNNING
-	RX bytes:0 acl:95 sco:0 events:208 errors:0
-	TX bytes:3488 acl:95 sco:0 commands:110 errors:0
-```
+## Working with Python CHIP Controller
 
-Then you can choose the adapter to use in command line arguments of the device
-controller:
+### 1. Prepare the Matter accessory.
 
-For example, add `--bluetooth-adapter=hci2` to use the virtual interface `hci2`
-listed above.
+In this tutorial, the [Matter: Light Bulb](/examples/lighting-app) example with Bluetooth LE commissioning will be used, but similar steps can be executed with the rest of the examples. Check [Matter examples](/examples) for the list of available samples.
 
-```
-chip-device-ctrl --bluetooth-adapter=hci2
-```
+Build and program the device with the Matter accessory firmware by following the example's documentation, for example [nRF Connect Lighting Example Application](/examples/lightigng-app/nrfconnect).
```

Comment:
Some people may not like this suggestion ;)

Reply:
Will remove it.